OpenSim/Common/Test/testOperators.cpp

```diff
@@ -0,0 +1,421 @@
+/* -------------------------------------------------------------------------- *
+ *                OpenSim:  testOperators.cpp                                 *
+ * -------------------------------------------------------------------------- *
```

Comment:
making this `testDelayOperator.cpp` would be more specific. Also a single .cpp will likely get unwieldy quickly if we test all the operators we are planning in here.